src/Analyzers/CSharp/CodeFixes/NamespaceSync/CSharpNamespaceSyncCodeFixProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixes.NamespaceSync;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.NamespaceSync
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.SyncNamespace), Shared]
+    internal sealed class CSharpNamespaceSyncCodeFixProvider : AbstractNamespaceSyncCodeFixProvider
+    {
+
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public CSharpNamespaceSyncCodeFixProvider()
+        {
+        }
+
+        protected override string Title => CSharpAnalyzersResources.Namespace_does_not_match_folder_structure;
```

Comment:
This feels like a common string for both C# and VB.  I don't see anything C# specific about it,

Reply:
this title also seems problematic.  it isn't clear to me what it's going to do.  Shouldn't this be more like: `Change namespace to {0}`, or `Move to folder {0}`?